pkgs/servers/mail/mailpit/update.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p nix coreutils curl jq nix-prefetch-git prefetch-npm-deps
+
+set -x -euo pipefail
+
+OWNER=axllent
+REPO=mailpit
+
```

Comment:
```suggestion
```

Reply:
I'd like to keep that as it's used multiple times (L9 and L34). So if the Repo would be moved in the future it only needs to be changed once and no occurrence can be missed.